### ISSUE TITLE
How about including push button handling?

### ISSUE THREAD

**Issue by wolfgangbregel** [miketeachman/micropython-rotary#21]
First of all, many thanks for your contribution. I tried quite a few solutions on encoder handling, yours is the most complete and stable one i found.

Talking about complete:
How about including push-button handling?
This should be an easy thing to do - and very handy.

Best regards
Wolf

**Comment by fiechr** [comment]
I agree, that would be convenient, although it is just a regular button in the end.

Is there maybe a recommended lib that already includes some software filtering/debounce?
